src/modules/job-manager/jobtap.c

```diff
@@ -505,6 +500,11 @@ struct jobtap *jobtap_create (struct job_manager *ctx)
         goto error;
     }
 
+    if (load_builtins (jobtap) < 0) {
```

Comment:
It would be nice to have a commit message for 706db0b507d1672724a018c14f39bdb1307124a7 for completeness.

Reply:
Style consistency comments are welcome BTW.

Reply:
A separate PR might be nice to get an extra entry in the release notes. However, I'd say it is up to you, if it causes any issues splitting this work up, then I'd say just push the doc and test updates to this PR.